tests/test_models.py

```diff
@@ -344,7 +344,7 @@ def test_functional_domain(functional_domains, gene_descriptors):
                 "associated_gene": gene_descriptors[0],
             }
         )
-    msg = "value is not a valid enumeration member; permitted: 'lost', 'preserved'"
+    msg = "Input should be 'lost' or 'preserved'"
```

Comment:
for the sake of getting this upgrade out the door, I just changed all of the expected messages to be the updated ones provided by pydantic. I don't love that we're checking hardcoded messages against their errors (since this is effectively testing their classes which I'm sure they already do :-) ).
Options:
1. we provide our own messages when these errors get raised and can test for those
2. techdebt and just update these later?
3. only check error type for the pydantic-specific errors thrown